.travis.yml

```diff
@@ -43,14 +43,16 @@ jobs:
     script:
     - GOOS=darwin GOARCH=amd64 go build -ldflags="$LINKFLAGS"
     - tar czf ${PROJECT_NAME}-${TRAVIS_TAG}-x86_64-apple-darwin.tgz wash
+    - tar czf ${PROJECT_NAME}-latest-x86_64-apple-darwin.tgz wash
```

Comment:
This could be a copy of the previous tar. I think it probably should be; somehow tar isn't necessarily deterministic.

Reply:
You're right. Updated.